Etabs_Adapter/ETABSAdapter.cs

```diff
@@ -79,6 +79,7 @@ public ETABS17Adapter(string filePath = "", EtabsSettings etabsSetting = null, b
         [Description("Creates an adapter to ETABS version 18 or later. For connection to ETABS v17 use the ETABS17Adapter. For connection to ETABS 2016 use the ETABS2016Adapter. Earlier versions not supported.")]
         public ETABSAdapter(string filePath = "", EtabsSettings etabsSetting = null, bool active = false)
 #endif
+        
```

Comment:
Why is this file being edited? Please remove.

Reply:
@peterjamesnugent blank line removed ;)